cmd/migration-manager/internal/cmds/source.go

```diff
@@ -329,6 +322,7 @@ func (c *cmdSourceUpdate) Run(cmd *cobra.Command, args []string) error {
 	newSourceName := ""
 	switch src.SourceType {
 	case api.SOURCETYPE_VMWARE:
+		vmwareProperties := api.VMwareProperties{}
 		err := json.Unmarshal(src.Properties, &vmwareProperties)
```

Comment:
Since you're populating the pre-existing source properties on update, if its previous connectivity status was OK, even if the authentication mode, credentials, or endpoint changed, we don't re-check connectivity, because `checkSourceConnectivity` skips any previously OK sources. Do I have that right?

Reply:
Yes, that's a good catch; at the moment I've updated it so it always resets the connectivity status on an update.